packages/scaffold/src/client.ts

```diff
@@ -327,6 +327,10 @@ export class Web3ModalScaffold {
     BlockchainApiController.setClientId(clientId)
   }
 
+  protected setQRCodeURI: (typeof ConnectionController)['setQRCodeURI'] = uri => {
```

Comment:
the uri is not only for the QR code but for all connections using WalletConnect protocol. maybe `setURI` or `setWalletConnectURI` works better?